Sources/CocoaLumberjack/DDLog.m

```diff
@@ -1062,17 +1070,14 @@ - (BOOL)isEqual:(id)other {
 }
 
 - (NSUInteger)hash {
-    return [super hash]
```

Comment:
Same here. Please add a quick comment on why super isn't called. Although for hashing it could be included, right?